src/pages/home/report/ReportActionCompose/listenKeyDown.js

```diff
@@ -0,0 +1,5 @@
+function listenKeyDown(callback) {
+    document.addEventListener('keydown', callback);
```

Comment:
@tienifr nitpick: style change -

condense `listenKeyDown` and `removeListenKeyDown` to a single file.